tests/test_data_types.py

```diff
@@ -14,7 +14,8 @@ def test_GetFeaturesResponse_from_dict(self):
         with open(file_1) as f:
             resp = json.load(f)
         resp = GetFeaturesResponse.from_response(resp)
-        self.assertEquals(resp.result.features, [["0"], None, [55.5, 57.88, 58.96, 57.66, None, 55.98]])
+        self.assertEquals(resp.result.features,
+                          [["0"], None, [55.5, 57.88, 58.96, 57.66, None, 55.98], ['0', '1', None, '3', '4', '5']])
```

Comment:
I'm guessing a formatter needs run to change the single quotes to double quotes. Are there precommit checks that apply the formatter?

Reply:
Oh already existsed, I just hadn't installed the pre-commit...